Documentation/api.md

```diff
@@ -407,6 +407,7 @@ PrometheusSpec is a specification of the desired behavior of the Prometheus clus
 | overrideHonorTimestamps | OverrideHonorTimestamps allows to globally enforce honoring timestamps in all scrape configs. | bool | false |
 | ignoreNamespaceSelectors | IgnoreNamespaceSelectors if set to true will ignore NamespaceSelector settings from the podmonitor and servicemonitor configs, and they will only discover endpoints within their current namespace.  Defaults to false. | bool | false |
 | enforcedNamespaceLabel | EnforcedNamespaceLabel enforces adding a namespace label of origin for each alert and metric that is user created. The label value will always be the namespace of the object that is being created. | string | false |
+| queryLogFile | QueryLogFile specifies the file to which PromQL queries are logged. This is only available in versions of Prometheus >= 2.16.0. | string | false |
```

Comment:
Let's comment that the user needs to ensure that the location is writable, and attach a volume if they want to persist it.

Reply:
Maybe even how to integrate the query log into the standard application log stream.